test/neo4clj/query_builder_test.clj

```diff
@@ -20,24 +20,49 @@
         "CREATE (G__123:Label1 {a: '1'})" node-complete false
         "CREATE (G__123:Label1 {a: '1'}) RETURN G__123" node-complete true))))
 
-(t/deftest lookup-query
-  (t/testing "Cypher for lookup query"
+(t/deftest lookup-node
```

Comment:
For both `lookup-query` and `lookup-relationship` I think it would be better to test the function (with the same name) in the `cypher` namespace. This avoids testing the same functionality with `return?` both `true` and `false`.

Instead test the functions here in the `query-builder` NS for prefixing with `"MATCH "` and the conditional `"RETURN <something>"`.

Reply:
I got the same thought when I wrote the new tests as they have a lot of overlapping.

Reply:
Fixed